react/week2/src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React, {useState, useEffect} from 'react'
+// import './ListItems.css';
+
+function Header() {
+
+const  [count, setCount] = useState(0)
+
+useEffect  ( () => {
+    document.title = `todo project`
+
+    var timer = setTimeout (() => {

```

Comment:
Your timer is recreated any time your state changes, whether the second passed, or not. It leads to the timer to not count seconds if you start to create a lot of todos or to drop them - it just hangs until you let the timeout to finish. To fix this issue, you can add a condition when you want your current timer to be cleared, and your next timer to be created. To do this, pass `[count]` as a second argument to `useEffect`. Also you could read some info here:

https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects